processor/probabilisticsamplerprocessor/metadata.yaml

```diff
@@ -28,3 +28,10 @@ telemetry:
       sum:
         value_type: int
         monotonic: true
+    processor_probabilistic_sampler_count_spans_sampled:
+      enabled: true
```

Comment:
this should not be enabled by default. That would constitute a breaking change.

Reply:
Fixed it. Thanks